src/main/java/org/candlepin/resource/OwnerResource.java

```diff
@@ -1018,14 +1018,17 @@ public Set<String> ownerServiceLevels(
     }
 
     @Override
-    public CandlepinQuery<ActivationKeyDTO> ownerActivationKeys(
+    public Iterable<ActivationKeyDTO> ownerActivationKeys(
```

Comment:
Just return the stream directly instead of recollecting it.

(note this applies several times throughout this PR)